packages/lambda-functions/template.yaml

```diff
@@ -12,27 +12,20 @@ Resources:
   HelloWorldFunction:
     Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
     Properties:
-      CodeUri: functions/hello-world
-      Handler: app.lambdaHandler
+      CodeUri: ../../dist/packages/lambda-functions
+      Handler: main.lambdaHandler
       Runtime: nodejs18.x
       Architectures:
         - x86_64
+      Environment:
+        Variables:
+          DATABASE_URL:
```

Comment:
We also need to define an environment variable for the database url

Reply:
Do I need to set a value here in testing? Or does that get set by `env.json`